Sources/SwiftProtobuf/api.pb.swift

```diff
@@ -35,17 +35,17 @@
 // THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 // (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-
 import Foundation
```

Comment:
Granted this is a generated file, but why lose the blank line between a comment (license) and then the code, this blank seems like it was useful because of the type of content.

Reply:
I think this is a problem with the rule that sorts the imports; it may not be preserving the leading trivia correctly on those lines. I need to overhaul the logic there anyway.